Mathlib/Analysis/Complex/UpperHalfPlane/Manifold.lean

```diff
@@ -70,4 +72,51 @@ lemma mdifferentiable_iff {f : ℍ → ℂ} :
     fun h ⟨z, hz⟩ ↦ mdifferentiableAt_iff.mpr <| (h z hz).differentiableAt
       <| (Complex.continuous_im.isOpen_preimage _ isOpen_Ioi).mem_nhds hz⟩
 
+lemma contMDiff_num (g : GL(2, ℝ)⁺) :
+    ContMDiff 𝓘(ℂ) 𝓘(ℂ) ω (num g) :=
```

Comment:
Instead of writing these just for omega, can you write them for a generic `{n : WithTop ℕ∞}` (that could/should be in the `variable` list)? The reason is that sometimes a lemma will need ContMDiff for n=1 or 2, and then the generic version spares you one step where you deduce this from the omega version.